### ISSUE TITLE
June 2023 Maintenance Sprint

### ISSUE THREAD

**Issue by VanessaVenti** [harvard-lts/CURIOSity#366]
Accessibility issues fixed during the June 2023 sprint. See related Jira ticket for more details: https://jira.huit.harvard.edu/browse/LTSMAINT-752

- [x] Option to switch language is not available to end users, it only displays when signed in. This is needed for the Coptic studies site for summer of 2023. Once working, many other sites could be translated as well, particularly CRB, Rubbings, IOHP, IHP, MPCOL who all have heavy user bases of non-native English speakers.
- [x] Filter rail - Focus state for metadata label covers the content in the first value. Also, should the focus state appear on click or just via keyboard? 
- [x] Linkify links in the Notes, [Source Institution Copy](https://curiosity.lib.harvard.edu/black-teacher-archive/catalog/153-GUT50000C02358) ([QA example](https://curiosity-qa.lib.harvard.edu/black-teacher-archive/catalog/145-GUT50000C00559)), and other fields when applicable on the item detail page. Getting this to work in the Source Institution Copy field is a high priority for BTA (May 2023). Info from DAS: According to [SC 2.4.4 Link Purpose (In Context) (Level A)](https://urldefense.proofpoint.com/v2/url?u=https-3A__www.w3.org_WAI_WCAG21_Understanding_link-2Dpurpose-2Din-2Dcontext.html&d=DwMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=7GPIuli0v7sUNtnBCpOe6aqO5lK-yQlOvzJV4eXtoFI&m=eJRqSRFnRMC4JVDT1xikxKnl6gl2RmCk-CEYz4I0_jd9xOGDpei7lPZoqS5pr88U&s=cOlSKYPHcx4ILSobX-SmKzemdCTPzRRbt76DEtfeZVU&e=), users should be able to understand the purpose or destination of a link to decide if they want to click it. By not actually hyperlinking the URL text, users who rely on assistive technologies such as screen readers may not be able to interact with the link or access its destination in an easy way. For example, unlinked URL text will not be identified when screen reader users search with keyboard shortcuts or open link lists. 
- [x] Add a skip viewer button (like skip to main content) so a nonsighted user can go to the data below. There are many random sequences of speech before a screenreader gets the title of the image in the image viewer.
- [x] Add numbers back to search results list. For long lists of results, makes it easier to jump back to a previous item you were looking at. We know researchers use these numbers. (Requested by Law on 2023-02-28.)
- [x] Add comma for results totals over 999. For example [this search in Women Working](https://curiosity-qa.lib.harvard.edu/women-working-1800-1930/catalog?exhibit_id=women-working-1800-1930&q=&search_field=all_fields) returns 4,429 results but it displays as 4429.
- [x] Add padding below each contact entry. Currently if you add more than one contact to a site there is no space between them making it difficult to distinguish them. 
  - [x] Expeditions requested to add a second contact in March 2023. Turn on display for both when fixed and notify contacts.
- [x] In gallery view, images don't have a focus state which makes it hard to understand which image you are on
- [x] In masonry view, you are unable to see the titles of the images on focus alone. There should be a way for a keyboard user to see the hover state that displays the title. So we want title based on focus state
- [x] Add helper text to "Download this item dropdown" for PDS objects to say you will be getting a PDF, not a high resolution image and to reach out to contact listed below for a high resolution image. This is a heavily used feature. (Requested by Botany on 2023-03-08)
- [x] There is a hidden heading 1 before the heading 2, the screenreader says it twice, this is for most pages. Some of the pages, just have it be a H2
